### PR TITLE
Add check_username request

### PR DESCRIPTION
**Pull request by doredesign**

### CODE REVIEW

File: spec/discourse_api/api/users_spec.rb [hunk 1]
```diff
@@ -284,4 +284,23 @@
...
+
+  describe "#check_username" do
+    let(:url) { "http://localhost:3000/users/check_username.json?username=sparrow&api_key=test_d7fd0429940&api_username=test_user" }
+
+    before do
+      body = '{"available":false,"suggestion":"sparrow1"}'
+      stub_get(url).to_return(body: body, headers: { content_type: "application/json" })
+    end
+
+    it "makes the correct delete request" do
```
Comment (oblakeerickson) [inline spec/discourse_api/api/users_spec.rb]: Looks like you need to update the text here.

Comment (doredesign) [reply spec/discourse_api/api/users_spec.rb]: Oof! Good catch.

Comment (doredesign) [reply spec/discourse_api/api/users_spec.rb]: @oblakeerickson fixed

**Comment by oblakeerickson** [comment]
Awesome, thanks!

**Pull request merged**